content/index.md

```diff
@@ -1,18 +1,22 @@
 ---
 title: OSRA Infrastructure Project
 ---
+Welcome to the Open Robotics Infrastructure Project Site.
 
-Welcome to the [OSRA](https://osralliance.org/) Infrastructure Project Site. This project aims to support the ROS and Gazebo ecosystem by providing essential infrastructure, including buildfarms, documentation, and tools for ROS and gazebo packages.
+The Infrastructure project is responsible for maintaining and improving the essential tools and systems that underpin the entire ROS and Gazebo ecosystem by providing essential infrastructure. This includes managing the build servers, continuous integration platforms, and various online resources that developers and users rely on to create and utilize robotics software. By ensuring the smooth operation of these critical components, the Infrastructure project plays a vital role in supporting the development and growth of the robotics community.
 
-This site is a work in progress and will be updated as the project progresses (and learn how to use Nanoc).
+# How to Contribute
 
 
-# How to Contribute
+Want to contribute to the OSRA Infrastructure project? 
+We welcome contributions of all kinds! 
+Whether you're a seasoned developer or a curious enthusiast, there are opportunities for everyone.
+Get involved by reporting issues, suggesting improvements, or diving into the codebase. 
 
+To learn more about how to contribute, check out our contribution guidelines on the project's 
+[GitHub repositories](/repos) or join our community forums for discussions and support: [discourse.ros.org](https://discourse.ros.org/), [OSRF Community Discord](https://discord.com/invite/RtJeHT8mXQ).
 
-To contribute to the OSRA Infrastructure Project, you can start by checking the follwing documents:
+If you would like to know more about the project, please check out the following resources:
 
   <a class="btn btn-primary" role="button" target="_blank" href="https://osralliance.org/staging/wp-content/uploads/2024/03/OSRA-explainer.pdf">OSRA Explainer</a>
   <a class="btn btn-primary" role="button" target="_blank" href="https://osralliance.org/staging/wp-content/uploads/2024/03/infrastructure_project_charter.pdf">Infrastructure Project Charter</a>
```

Comment:
Use markdown bullets instead of buttons.

```suggestion
* [The Open Source Robotics Alliance (OSRA) Explained](https://osralliance.org/staging/wp-content/uploads/2024/03/OSRA-explainer.pdf)
* [Infrastructure project charter](https://osralliance.org/staging/wp-content/uploads/2024/03/infrastructure_project_charter.pdf)
```